chapters/planning-data-work.tex

```diff
@@ -1,60 +1,53 @@
+Preparation for collaborative data work begins long before you acquire any data,
+and involves planning both the software tools you will use
 and the collaboration platforms and processes for your team.
 In order to be prepared to work on the data you receive with a group,
-you need to plan out the structure of your workflow in advance.
+you need to structure your workflow in advance.
 This means knowing which data sets and output you need at the end of the process,
 how they will stay organized, what types of data you'll handle,
 and whether the data will require special handling due to size or privacy considerations.
-Identifying these details should help you map out the data needs for your project,
-giving you and your team a sense of how information resources should be organized.
-It's okay to update this map once the project is underway --
-the point is that everyone knows -- at any given time -- what the plan is.
+Identifying these details will help you map out the data needs for your project,
+and give you a sense for how information resources should be organized.
+It's okay to update this data map once the project is underway.
+The point is that everyone knows -- at any given time -- what the plan is.
 
-To implement this plan, you will need to prepare collaborative tools and workflows.
+To do data work effectively in a team environment,
+you will need to prepare collaborative tools and workflows.
 Changing software or protocols halfway through a project can be costly and time-consuming,
-so it's important to think ahead about decisions that may seem of little consequence.
-For example, things as simple as sharing services, folder structures, and filenames
-can be extremely painful to alter down the line in any project.
+so it's important to plan ahead.
+Seemingly small decisions such as sharing services, folder structures,
+and filenames can be extremely painful to alter down the line in any project.
 Similarly, making sure to set up a self-documenting discussion platform
-and version control processes
-makes working together on outputs much easier from the very first discussion.
-This chapter will discuss some tools and processes that
-will help prepare you for collaboration and replication.
-We will provide free, open-source, and platform-agnostic tools wherever possible,
-and point to more detailed instructions when relevant.
-(Stata is the notable exception here due to its current popularity in the field.)
-Most have a learning and adaptation process,
-meaning you will become most comfortable with each tool
-only by using it in real-world work.
-Get to know them well early on,
-so that you do not spend a lot of time learning through trial and error.
+and process for version control;
+this makes working together on outputs much easier from the very first discussion.
+This chapter will guide you on preparing a collaborative work environment,
+and structuring your data work to be well-organized and clearly documented.
+
```

Comment:
This seems to be the only intro that does not include a brief description of the chapter. Is there a particular reason for that?

Reply:
@mariaruth

Reply:
Done in 96e2709df3eea881c39d04b5d723aeac8e87714b